Makefile

```diff
@@ -114,6 +114,8 @@ install: install-mk libdragon
 	install -Cv -m 0644 include/rspq.h $(INSTALLDIR)/mips64-elf/include/rspq.h
 	install -Cv -m 0644 include/rspq_constants.h $(INSTALLDIR)/mips64-elf/include/rspq_constants.h
 	install -Cv -m 0644 include/rsp_queue.inc $(INSTALLDIR)/mips64-elf/include/rsp_queue.inc
+	install -Cv -m 0644 src/fatfs/diskio.h $(INSTALLDIR)/mips64-elf/include/diskio.h
+	install -Cv -m 0644 src/fatfs/ff.h $(INSTALLDIR)/mips64-elf/include/ff.h
```

Comment:
Maybe we could install those into a fatfs subdir to separate them. Not a big deal but I think it would be a tad neater